applications/MappingApplication/MappingApplication.py

```diff
@@ -24,6 +26,9 @@ def GetRegisteredMPIMapperNames(*args):
     from KratosMultiphysics.MappingApplication.MPIExtension import MPIMapperFactory
     return MPIMapperFactory.GetRegisteredMapperNames(*args)
 
+MapperFactory = CoreMapperFactory
+Mapper = CoreMapper
```

Comment:
Why not just:

```Python
from KratosMultiphysics import Mapper
from KratosMultiphysics import MapperFactory
```

Seems like a double indirection as it is right now. Are the `CoreXXX` alias used or planed to be used anywhere?

Reply:
good point, not sure why I didn't do it in that way...